Documentation~/CinemachineBodyTransposer.md

```diff
@@ -8,7 +8,7 @@ The fixed offset can be interpreted in various ways, depending on the Binding Mo
 
 | **Property:** || **Function:** |
 |:---|:---|:---|
-| __Binding Mode__ || The coordinate space to use when interpreting the offset from the target, and the damping. For example, for a "Door Cam or “Hood Cam" in a car racing game, use Local Space Locked To Target. |
+| __[Binding Mode](CinemachineBindingModes.md)__ || The coordinate space to use when interpreting the offset from the target, and the damping. For example, for a "Door Cam or “Hood Cam" in a car racing game, use Local Space Locked To Target. |
```

Comment:
There is also a Binding Mode property in Orbital Transposer - need a similar change there

Reply:
FreeLook also

Reply:
Linked added to those topics.